### PR TITLE
Add database & password parameters support for predis sentinel

### PR DESCRIPTION
**Pull request by Erliz**
If you use Sentinel you could not set db id in dsn like this `redis://%redis_host%:%redis_port%/1` you will have an exception:
```
[Predis\ClientException]
  No sentinel server available for autodiscovery.
```
cause this dsn is connection for sentinel hosts and it have no idea about database number.
For this reason predis add additional `parameters` to connection options
in this issue https://github.com/nrk/predis/issues/131#issuecomment-219053799
by this commit https://github.com/nrk/predis/commit/a816adf6e7c407d70dec8447d2115ee39a9cce29

After a day of debugging this bundle and predis i found this magic method `setDefaultParameters` and it's work like a charm for me.

i think it will be relative for 
#274 
#315
#339
#336

### CODE REVIEW

**Comment by MyDigitalLife** [comment]
It would be nice to also update the documentation in Resources/doc/index.md to reflect that these options have been added.

**Comment by Erliz** [comment]
I already add it [doc_update](https://github.com/snc/SncRedisBundle/pull/340/files#diff-0b18810f2df707500fc6ea31f0d4f897)

**Comment by Erliz** [comment]
After sentinel return connection to master/slave there is no option to wrap `NodeConnection` with `ConnectionWrapper`, i add it similarly by sentinel connection.

**Comment by MyDigitalLife** [comment]
Sorry, didn't see that. Nice job.

**Comment by Crozin** [comment]
Any chance for merge anytime soon?

File: Resources/doc/index.md [hunk 1]
```diff
@@ -134,9 +134,13 @@ snc_redis:
             options:
                 replication: sentinel
                 service: mymaster
+                parameters:
+                    database: 1
+                    password: pass
 ```
 
 The `service` is the name of the set of Redis instances.
+The `parameters` is optional parameters that added to master/slave connections and ignores by sentinel connections.
```
Comment (snc) [inline Resources/doc/index.md]: Can you please change this to:

The optional `parameters` option can be used to set parameters like the database number and password for the master/slave connections, they don't apply for the connection to sentinal.

(if it sounds correct to you, I don't have sentinal experience)

Comment (Erliz) [inline Resources/doc/index.md]: Improved

**Comment by snc** [comment]
Thanks, merged.

**Pull request merged**